### DISCUSSION TITLE
Is the much more output size since updating to v5.7.0 using PySide6 in a project normal?

### DISCUSSION THREAD

**Discussion by BlockyTheDev** [pyinstaller/pyinstaller#7322] [PyInstaller] [answered]
(File sizes read from Windows Folder Properties Popup)
v5.6.2:
=> Project files included: ~90 KB
=> Python 3.11
=> Modules: colorame, requests, PySide6
=> Output file size (Built as directory): ~ 100MB

v5.7.0:
=> Project files included: ~90 KB (Same as above)
=> Python 3.11
=> Modules: colorame, requests, PySide6
=> Output file size (Built as directory): ~ 250MB

What I am seeing is that in v5.7.0 there are many QTWebEngine files included like a file with the name `QtWebEngineCore` with a size of ~130MB, which wasn't included in v5.6.2, which I think isn't needed in this project, because QTWebEngine isn't used (When I remove the dll file my projects works without problems, too.)

Now my questions: Is this normal and why are these for me not needed files included?

**Comment by rokm** [comment]
What PySide6 modules are you using?

**Comment by BlockyTheDev** [comment, marked as answer]
I am importing following things:
```python
from PySide6.QtCore import QFile, QIODevice
from PySide6.QtGui import QIcon
from PySide6.QtUiTools import QUiLoader
from PySide6.QtWidgets import (QApplication, QComboBox, QFileDialog, QLineEdit,
                               QPlainTextEdit, QPushButton, QToolButton,
                               QWidget)
```

Installed I have `PySide6-Essentials, PySide6-Addons, shiboken6, PySide6` from PyPi, but I think they are installed all while installing the normal `PySide6` module

**Reply by rokm** [reply]
> from PySide6.QtUiTools import QUiLoader

I suppose it is "normal", then. Because in 5.7.0, we now have a hook for `QtUiTools` that ensures that `designer` plugins are collected, and among those is a `qwebengineview` plugin that depends on `QtWebEngine`. 

Although perhaps we should tie those only to the `QtDesigner` module.

https://github.com/pyinstaller/pyinstaller/blob/3657c7c2937d055a8622d61ab559c768a3c72fe2/PyInstaller/utils/hooks/qt/_modules_info.py#L369-L371

If you remove "designer" from this plugins list in your local copy of PyInstaller 5.7.0, does the size go back to normal?

**Reply by rokm** [reply]
> Although perhaps we should tie those only to the `QtDesigner` module.

I think that will be for the best. I've opened #7324 with this change.

**Reply by BlockyTheDev** [reply]
I installed the version from your PR from source and know the file size went back to around 116MB, which is normal through the module updates I did after the first build.
